reducers.md

```diff
@@ -735,6 +735,10 @@
   https://github.com/lundegaard/redux-extensible-store  
   Redux extension that allows you to dynamically inject new reducers and sagas into you running application.
   
+- **redux-dynamic-modules**
+  https://github.com/Microsoft/redux-dynamic-modules
+  Redux library which helps you organize all of your Redux artifacts (middleware, reducers, sagas, thunks, epics) into modules and dynamically add those modules at runtime
```

Comment:
Redux library designed for code-splitting by dynamically loading reducers, middlewares, sagas, thunks You can define a module, which contains reducers and middleware, and add it to the Redux store at runtime. We also provide a HOC `DynamicModuleLoader`, which can load/unload modules on mount/unmount.